src/kernels/kernelFinder.base.ts

```diff
@@ -98,10 +98,6 @@ export abstract class BaseKernelFinder implements IKernelFinder {
             }),
             this.listRemoteKernels(resource, cancelToken, useCache).catch((ex) => {
                 traceError('Failed to get remote kernels', ex);
-                // When remote kernels fail, turn off remote if we get a ECONNREFUSED error
-                if (ex.toString().toLowerCase().includes('econn')) {
```

Comment:
This should be removed for #9167, as the user will decide what to do with old (busted) remotes.
I.e. if the remote is invalid, we log the errors, and if things have been cached, then we'll display them and user will be notified when they attempt to run a cell, at which point they can chose to remove that connection or switch to local or the like.